src/test/resources/test_loaded_resolv.conf

```diff
@@ -0,0 +1,5 @@
+# comment
+nameserver 192.168.1.1
+domain domain.com
+search example.com dnsjava.org
+options ndots:5
```

Comment:
Interesting. I wasn't aware there were this many options for resolv.conf. I wonder if we want to support some of them in the future (not for this PR, just a remark).

Reply:
Logged issue #57